docs/apis/data_en.md

```diff
@@ -0,0 +1,205 @@
+
+### `Transform`
+
+The `__call__` method of the `Transform` object takes a unique argument `sample`. `sample` must be a dictionary or a sequence of dictionaries. When `sample` is a sequence, perform data transformations for each dictionary in `sample` and return the results sequentially stored in a Python build-in list; When `sample` is a dictionary, the `Transform` object extracts input from some of its key-value pairs (these keys are called "input keys"), performs the transformation, and writes the results as key-value pairs into `sample`(these keys are called "output keys"). It should be noted that many of the `Transform` objects in PaddleRS currently have a carbon copy behavior, that is, an intersection between the input key and the output key. The common key names in `sample` and their meanings are as follows:
+
+|Key Name|Description|
+|----|----|
+|`'image'`|Image path or data. For change detection task, it refers to the first phase image data.|
+|`'image2'`|Second phase image data in change detection task.|
+|`'image_t1'`|First phase image path in change detection task.|
+|`'image_t2'`|Second phase image path in change detection task.|
+|`'mask'`|Truth label path or data in image segmentation/change detection task.|
+|`'aux_masks'`|Auxiliary label path or data in image segmentation/change detection tasks.|
+|`'gt_bbox'`|Detection box labeling data in object detection task.|
+|`'gt_poly'`|Polygon labeling data in object detection task.|
+|`'target'`|Target image path or data in image restoration task.|
+
+### Combined Data Transformation Operator
+
+Use `paddlers.transforms.Compose` to combine a set of data transformation operators. `Compose` receives a list input when constructed. When you call `Compose`, it serially execute each data transform operators in the list. The following is an example:
+
+```python
+# Compose a variety of transformations using Compose.
+# The transformations contained in Compose will be executed sequentially in sequence
+train_transforms = T.Compose([
+    # Read Image
+    T.DecodeImg(),
+    # Scale the image to 512x512
+    T.Resize(target_size=512),
+    # Perform a random horizontal flip with a 50% probability
+    T.RandomHorizontalFlip(prob=0.5),
+    # Normalized data to [-1,1]
+    T.Normalize(
+        mean=[0.5, 0.5, 0.5], std=[0.5, 0.5, 0.5]),
+    # Select and organize the information that needs to be used later
+    T.ArrangeSegmenter('train')
+])
+```
+
+Generally, in the list of `Compose` object accepted data transform operators, the first element is `paddlers.Transforms.DecodeImg` object, used to read image data; The last element is [`Arrange` Operator](https://github.com/PaddlePaddle/PaddleRS/blob/develop/paddlers/transforms/operators.py, used to extract and arrange information from the `sample` dictionary.
```

Comment:
in the list of `Compose` object accepted data transform operators -> in the list of data transform operators accepted by a `Compose` object